src/components/NavigationAppBar.js

```diff
@@ -154,7 +203,9 @@ class NavigationAppBar extends React.Component {
 }
 
 NavigationAppBar.propTypes = {
-  onChangePage: PropTypes.func.isRequired
+  classes: PropTypes.object.isRequired,
```

Comment:
eslint typically complains about this syntax.

Reply:
Yeah, unfortunately for "classes" it's really anonymous type since there is no telling what classes are there in the styles :( All Google gave me was this propType definition >*>